jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/TopicDraft.java

```diff
@@ -32,27 +31,24 @@
 }, message = "{topicDraft.fields.not_null}")
 public class TopicDraft extends Entity {
 
-    @Size(min = Topic.MIN_NAME_SIZE, max = Topic.MAX_NAME_SIZE,
+    @Size(min = 1, max = Topic.MAX_NAME_SIZE,
```

Comment:
A thought occurred to me.. Why don't we always have a draft for a person? Even if it's empty. Would that simplify our design?